src/App.js

```diff
@@ -6,7 +6,12 @@ import {
 	ProtectedRoute,
```

Comment:
![image](https://user-images.githubusercontent.com/67889207/168729197-8d7cac34-d857-42ed-b741-159020e41e3f.png)
When the posts are empty(following 0 accounts) show something so user knows that he should he needs to follow someone. the floating action button is very big maker little smaller and more on the right side.

Reply:
Thankyou, will work on UI-UX. Have already merged in local